spec/api/v1/resources_spec.rb

```diff
@@ -22,6 +23,84 @@
     end
   end
 
+  context 'publish on behalf without token' do
+    before do
+      resource_json = File.read('spec/support/fixtures/json/ce_registry/credential/1_valid.json')
+
+      organization = create(:organization)
+
+      post "/resources/organizations/#{organization.id}/documents",
+           resource_json
+    end
+
+    it 'returns a 401 unauthorized http status code' do
+      expect_status(:unauthorized)
+    end
+  end
+
+  context 'publish on behalf with token, can publish on behalf of organization' do
+    before do
+      resource_json = File.read('spec/support/fixtures/json/ce_registry/credential/1_valid.json')
+
+      organization = create(:organization)
+      create(:organization_publisher, organization: organization, publisher: user.publisher)
+
+      post "/resources/organizations/#{organization.id}/documents",
+           resource_json, 'Authorization' => 'Token ' + user.auth_tokens.first.value
```

Comment:
"Token #{user.auth_tokens.first.value}"

Also I'm not sure if this piece `user.auth_tokens.first.value` is used (or will be) on more places, but I guess each user has only one active token at a time, so we can have a method `auth_token` on the user model that encapsulates this. 
If it's only here, then nevermind.